### DISCUSSION TITLE
Pre-release always opens to a small window, not maximized

### DISCUSSION THREAD

**Discussion by ltsnow** [sumatrapdfreader/sumatrapdf#3942] [General]
With the latest pre-release 15919 Sumatra always opens to a small window, not the maximized one of 3.5.2.  The setting for WindowState is set to 2 in both.

**Comment by ltsnow** [comment]
Here's the deal.  If I just open the program and maximize the window it will indeed subsequently open maximized. Also it will stay maximized if I open a pdf file from WITHIN the program.  However, this is not how I use it.  I set Sumatra as my default program and if I open a pdf file from outside the program it will always open tall and narrow.  Even if you maximize the window the next time you open the file from OUTSIDE the program it is tall and narrow.  This is the first time this has happened and everything works fine with 3.5.2 and all previous versions.

**Reply by GitHubRulesOK** [reply]
You opened an issue https://github.com/sumatrapdfreader/sumatrapdf/issues/3945 these comments need to be there so am quoting from here.
